runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -234,7 +233,16 @@ public static class CorfuRuntimeParameters extends RuntimeParameters {
          */
         Duration holeFillTimeout = Duration.ofSeconds(10);
 
-        Duration mvoCacheExpiry = Duration.ofMinutes(10);
+        /*
+         * Cache expiry duration for the MVOCache of disk-backed tables.
+         */
+        Duration mvoCacheExpiryDiskBacked = Duration.ofMinutes(10);
+
+        /*
+         * Cache expiry duration for the MVOCache of in-memory tables.
+         */
+        long mvoCacheExpirySeconds = 0;
```

Comment:
nit: change type to Duration and rename to mvoCacheExpiryInMemory? so that these two are consistent.